addons/variable-manager/userscript.js

```diff
@@ -85,18 +85,32 @@ export default async function ({ addon, console, msg }) {
       this.scratchVariable = scratchVariable;
       this.target = target;
       this.visible = false;
+      this.ignoreTooBig = false;
       this.buildDOM();
     }
 
     updateValue(force) {
       if (!this.visible && !force) return;
+
       let newValue;
+      let maxSafeLength;
       if (this.scratchVariable.type === "list") {
         newValue = this.scratchVariable.value.join("\n");
+        maxSafeLength = 5000000;
       } else {
         newValue = this.scratchVariable.value;
+        maxSafeLength = 1000000;
```

Comment:
If the value can be viewed by clicking on it, I think the limit could probbably be one tenth this size. The current one is fine though.